sbnci/PlottingScripts/CompareDataDistributions.C

```diff
@@ -407,16 +422,19 @@ void CompareDataDistributions(TString gCurVersion="v07_06_00", TString gRefVersi
           bottomPad->cd();
           TH1D *ratioPlotFile2 = (TH1D*)hRef->Clone("ratioPlotFile2");
           ratioPlotFile2->Add(hRef, -1);
-          ratioPlotFile2->GetYaxis()->SetRangeUser(-10,10);
           ratioPlotFile2->Divide(hRef);
-          setStyleRatio(ratioPlotFile2, gRefVersion.Data(), gCurVersion.Data());
-          ratioPlotFile2->Draw("hist");
           TH1D* ratioPlotFile2C = (TH1D*)ratioPlotFile2->Clone("ratioPlotFile2C");
           ratioPlotFile2C->SetFillColor(0);
-          ratioPlotFile2C->Draw("histsame");
           TH1D *ratioPlotFile1 = (TH1D*)hCur->Clone("ratioPlotFile1");
           ratioPlotFile1->Add(hRef, -1);
           ratioPlotFile1->Divide(hRef);
+	  double maxvalue, minvalue;
+	  getHistMaxMinWithError(ratioPlotFile1, maxvalue, minvalue);
+	  double range = 1.2 * std::max(std::abs(maxvalue), std::abs(minvalue));
+          ratioPlotFile2->GetYaxis()->SetRangeUser(-range, range);
```

Comment:
Here maybe you want to use the actual range `[1.2*minvalue; 1.2*maxvalue]` rather than a symmetric range based on the max abs value of the two values?
`ratioPlotFile2->GetYaxis()->SetRangeUser(1.2*minvalue, 1.2*maxvalue);`